client/src/test/java/com/paypal/selion/TestServerListener.java

```diff
@@ -34,7 +37,7 @@ public void onFinish(ISuite suite) {
         try {
             TestServerUtils.stopServer();
         } catch (Exception e) { // NOSONAR
-            e.printStackTrace();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [System.err.println is used](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1529805945/issues/source?bid=2928425&fileBranchId=2989551#l40)